tests/ut/runtime/test_task_submit_dispatch.py

```
@@ -0,0 +1,298 @@
+    assert passed is False
+    assert "do not pass --execute-via-task-submit" in error
+
+
+# ---------------------------------------------------------------------------
+# _fused_execute_task dispatch — sim guard
+# ---------------------------------------------------------------------------
+
+
+def _artifact(platform):
+    return test_runner.CompileArtifact(
+        work_dir=Path("/tmp/x"),
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/x"

(S108)